cli/command/container/update.go

```diff
@@ -66,6 +67,7 @@ func NewUpdateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.Var(&opts.memorySwap, "memory-swap", "Swap limit equal to memory plus swap: '-1' to enable unlimited swap")
 	flags.Var(&opts.kernelMemory, "kernel-memory", "Kernel memory limit")
 	flags.StringVar(&opts.restartPolicy, "restart", "", "Restart policy to apply when a container exits")
+	flags.Int64Var(&opts.pidsLimit, "pids-limit", 0, "Tune container pids limit (set -1 for unlimited)")
```

Comment:
Added "annotation" for "pids-limit" with version 1.30.

@thaJeztah do you want me to change annotation version for "cpus"? Not sure why your code snippet has "cpus" with version 1.30, when the same line exists with 1.29 😅